addons/field-guide/addon/routes/show.js

```diff
@@ -29,6 +29,8 @@ export default class ShowRoute extends Route {
         return {
           id: res.data.id,
           ...res.data.attributes,
+          // TODO! TEMPORARY, FOR TESTING!
+          hasSidecar: true,
```

Comment:
Just curious, why do we call this a "sidecar" vs. "sidebar"? (Just never heard it called that before.)

Reply:
It's the name used for this "thing" in the DevDot/Developer project (https://developer.hashicorp.com/terraform - "on this page"). It's nice and funny and memorable, so we decided to keep it :) 

![](https://media.giphy.com/media/dXjLrEDc5XVisjgPu2/giphy.gif)